scripts/histmakers/mz_dilepton.py

```diff
@@ -188,6 +190,10 @@ def build_graph(df, dataset):
     axes = nominal_axes
     cols = nominal_cols
 
+    if args.addLumiAxis and dataset.is_data:
+        axes = [*axes, hist.axis.Regular(11, 0, 11, name = "lumi", underflow=False, overflow=False)]
```

Comment:
Any reason this can't be a Variable axis with the actual run ranges? (which could then be filled directly using the run number)